model/src/form/form-metadata/index.ts

```diff
@@ -28,11 +33,31 @@ export const teamEmailSchema = Joi.string()
  * Joi schema for `FormMetadata` interface
  * @see {@link FormMetadata}
  */
-export const formMetadataSchema = Joi.object<FormMetadata>()
+export const formMetadataCreateSchema = Joi.object<FormMetadata>()
```

Comment:
But then this type generic needs updating too

How about we name the schema to match the type too?

~`formMetadataCreateSchema`~ → `formMetadataInputSchema`

```suggestion
export const formMetadataInputSchema = Joi.object<FormMetadataInput>()
```

Reply:
Resolved with 3bbb93aa4f770999ea2292070f7183da5e99b323